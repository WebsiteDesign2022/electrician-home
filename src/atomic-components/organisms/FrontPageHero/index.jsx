import React from 'react';

import FrontPageHeroDescription from '../../molecules/FrontPageHeroDescription';

import './index.scss';

function FrontPageHero() {
  return (
    <div className="col-xxl-8 px-4 py-5 bg-dark">
      <div className="row flex-lg-row-reverse align-items-center g-5 py-5">
        <div className="col-10 col-sm-8 col-lg-6">
          <img src="bootstrap-themes.png" className="d-block mx-lg-auto img-fluid" alt="Bootstrap Themes" width="700" height="500" loading="lazy" />
        </div>

        <FrontPageHeroDescription />

      </div>
    </div>
  );
}

export default FrontPageHero;
