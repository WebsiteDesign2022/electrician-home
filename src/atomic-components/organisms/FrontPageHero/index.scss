#box-shadow-button-top-left {
    padding: 10px 10px 10px 10px !important;
    box-shadow: 10px 10px #0E2E74;
}

.box-shadow-bottom-right {
    padding: 10px;
    box-shadow: 10px 10px #4A4A4A;
}
