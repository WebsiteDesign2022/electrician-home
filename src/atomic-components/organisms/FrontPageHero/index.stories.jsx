import React from 'react';

import FrontPageHero from '.';

export default {
  title: 'Electrician-Home/organisms/SectionDivider',
  component: FrontPageHero,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <FrontPageHero {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
