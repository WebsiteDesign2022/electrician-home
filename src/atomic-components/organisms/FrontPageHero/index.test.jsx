import TestRenderer from 'react-test-renderer';

import FrontPageHero from '.';

describe("FrontPageHero", () => {
    test("renders FrontPageHero", () => {
        const frontPageHero = TestRenderer.create(
            <FrontPageHero/>
        );

        expect(frontPageHero.toJSON()).toMatchSnapshot();
    });
});
