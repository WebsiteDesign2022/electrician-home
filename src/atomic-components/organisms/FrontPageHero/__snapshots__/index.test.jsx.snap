// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrontPageHero renders FrontPageHero 1`] = `
<div
  className="col-xxl-8 px-4 py-5 bg-dark"
>
  <div
    className="row flex-lg-row-reverse align-items-center g-5 py-5"
  >
    <div
      className="col-10 col-sm-8 col-lg-6"
    >
      <img
        alt="Bootstrap Themes"
        className="d-block mx-lg-auto img-fluid"
        height="500"
        loading="lazy"
        src="bootstrap-themes.png"
        width="700"
      />
    </div>
    <div
      className="col-lg-6 justify-content-left"
    >
      <div
        className="arrow-top-left-orange"
      />
      <h1
        className="display-5 lh-1 mb-3 text-light"
      >
        Electrical Contractor
      </h1>
      <h1
        className="display-5 fw-bold lh-1 mb-3 text-light"
      >
        Licensed & Insured
      </h1>
      <div
        className="d-grid gap-2 d-md-flex justify-content-md-start"
      >
        <button
          className="btn btn-primary btn-lg px-10 me-md-2 rounded-0"
          id="box-shadow-button-top-left"
          type="button"
        >
          ABOUT US
        </button>
        <button
          className="btn btn-light text-primary btn-lg px-4 rounded-0 box-shadow-bottom-right"
          type="button"
        >
          CONTACT US
        </button>
      </div>
    </div>
  </div>
</div>
`;
